service/src/main/java/org/cbioportal/genome_nexus/service/EnsemblService.java

```diff
@@ -27,4 +28,5 @@ EnsemblTranscript getCanonicalEnsemblTranscriptByHugoSymbol(String hugoSymbol, S
 
     List<EnsemblTranscript> getEnsemblTranscripts(String geneId, String proteinId, String hugoSymbol);
     List<EnsemblTranscript> getEnsemblTranscripts(List<String> transcriptIds, List<String> geneIds, List<String> proteinIds, List<String> hugoSymbols);
+    HashMap<String, String> getHugoSymbolToEntrezGeneIdMap();
```

Comment:
Similar to the repository interface let's add a method like `getEntrezGeneIdByHugoSymbol(String hugoSymbol)`